Containerfile

```diff
@@ -2,9 +2,9 @@ FROM registry.access.redhat.com/ubi9/python-39
 
 USER root
 
-ENV GO_VERSION=1.19
+ENV GO_VERSION=1.20.3
```

Comment:
I think we could probably keep the patch version unpinned, so to get newer patches as they come out.

Reply:
It does not do that. It will download 1.20.0

Reply:
I thought otherwise. Ok fair enough then.